src/pages/checkIn.js

```diff
@@ -16,7 +16,7 @@ export const checkInTemplate = async (data) => {
     let collections = [];
     let visitCollections = [];
 
-    if(response.code != 404) {
+    if (response.data.length > 0) {
```

Comment:
All of these look good, my only question would be do we think we should add anything that looks at the response code itself? Like if the response code isn't `200` should we be acting differently?

Reply:
I think this is a general question to many of our UI components that depend on fetched data. 
I'm not sure whether an alert is needed to let the user know when there's an error, or silently ignore errors ( so that users are not bothered).